docs/pages/collectorintro.md

```diff
@@ -0,0 +1,35 @@
+
+#Introduction
+
+Refocus makes it easy for users to understand the real-time status of their systems. To date, achieving this has required users to develop API integrations in order to push Samples into Refocus. This process had a few limitations: it required developers to write API integrations, and, at times, meant teams had to operate an extra 'always-on' service. 
+
+Collectors are designed to tackle these issues and make it easier than ever to leverage all the capabilities of Refocus!
+
+#So, what is a Collector 
```

Comment:
needs question mark

Reply:
Why? None of the titles have ending punctuation.

Reply:
An interesting take on not to adding a period to the end of a title applies to this too I think. https://english.stackexchange.com/questions/32884/should-i-use-a-period-at-the-end-of-a-headline